src/libcrun/container.c

```diff
@@ -368,12 +368,23 @@ static char *potentially_unsafe_annotations[] = {
 #define SYNC_SOCKET_MESSAGE_LEN(x, l) (offsetof (struct sync_socket_message_s, message) + l)
 
 static int
-sync_socket_write_msg (int fd, bool warning, int err_value, const char *log_msg)
+sync_socket_write_msg (int fd, int verbosity, int err_value, const char *log_msg)
 {
   int ret;
   size_t err_len;
   struct sync_socket_message_s msg;
-  msg.type = warning ? SYNC_SOCKET_WARNING_MESSAGE : SYNC_SOCKET_ERROR_MESSAGE;
+  switch (verbosity)
+    {
+    case LIBCRUN_VERBOSITY_DEBUG:
+      msg.type = SYNC_SOCKET_SYNC_MESSAGE;
```

Comment:
shall we define `SYNC_SOCKET_DEBUG_MESSAGE`?

Reply:
Yes makes sense, I added it as suggested.